### ISSUE TITLE
sentinel mobs lose their inherent AI

### ISSUE THREAD

**Issue by Volkenstein666** [mcmonkey4eva/Sentinel#89]
creeper sentinels dont explode - blaze sentinels float up to you and bump into you instead of hurling fireballs - something in config im missing? this isnt a citizens issue I never had this issue with sentry

**Comment by mcmonkey4eva** [comment]
If you want default AI; use default mobs. If you want custom AI, use NPCs equipped with the relevant item in Sentinel. There is a fireball attack for Sentinel NPCs, but there's no self destruct one.

**Comment by Volkenstein666** [comment]
I didnt realize non humanoid mobs could be equipped with the items that trigger a special attack. Ill give it a try

**Comment by Volkenstein666** [comment]
mc I just realized you were the guy keeping Sentry alive. What changed in the minecraft code to cause these changes in the plugin? Mobs that are sentinels - for instance iron golems or zombies - have no animations when they attack anything - they just bump into them - the golems use to swing their arms when they attacked- I worked with Sentries extensively I remember what it used to look like. The fireballs that were shot from blaze rods used to fly through the air on their way to the target like ghast fireballs- these fireballs hit like lightning. they hit you as soon as theyre cast. Eyes of ender no longer bounce people in the air. Witches dont throw potions- they may throw then once but if they are injured whatever they were equipped with gets switched off once they drink their healing potion. I tend to blame mojang for doing everything in their power to pretend the modding community doesnt exist but something must have happened for the plugin to be stripped down like this.
